.github/workflows/pr-check.yaml

```diff
@@ -319,13 +322,30 @@ jobs:
       - name: Execute pnpm
         run: pnpm install
 
+      - name: Build Podman Desktop for E2E tests Production Mode
+        if: ${{ matrix.mode == 'production' }}
+        env:
+          ELECTRON_ENABLE_INSPECT: true
+        run: |
+          pnpm compile:current --linux dir
+          path=$(realpath ./dist/linux-unpacked/podman-desktop)
+          echo "Podman Desktop built binary: $path"
+          echo "PODMAN_DESKTOP_BINARY_PATH=$path" >> $GITHUB_ENV
+
       - name: Run E2E smoke tests
+        if: ${{ matrix.mode == 'development' }}
```

Comment:
the if is not possible just to add the env ?

Reply:
Should be.